sections/migrations.js

```diff
@@ -162,6 +168,16 @@ export default [
       $ knex migrate:down
     `
   },
+  {
+    type: "text",
+    content: "To List Both Completed and Pending Migrations"
```

Comment:
Capitalization is weird here, can you make it a normal sentence casing?
Also `:` is missing in the end,